content/en/platform/corda/4.8/enterprise/notary/faq/notary-failover.md

```diff
@@ -13,10 +13,11 @@ title: Notary Failover
 
 # Notary Failover
 
-Corda has built in features that would retry flows on specific cases of failure. See node-flow-hospital for information
-that applies to all flows.
+Corda has built-in features to allow for the retrying of flows in specific cases of failure.
+See [node-flow-hospital](content/en/platform/corda/4.8/enterprise/node/node-flow-hospital.md) for information that applies
+to all flows.
 
-The FinalityFlow contains notary specific logic which can initiate a backpressure-aware subflow since minimum platform version 4.
+The FinalityFlow contains notary specific logic which can initiate a backpressure-aware sub-flow since minimum platform version 4.
```

Comment:
```suggestion
The FinalityFlow contains notary-specific logic which can initiate a backpressure-aware sub-flow since minimum platform version 4.
```